src/migration/1682497694917-FreeformPost.ts

```diff
@@ -0,0 +1,18 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class FreeformPost1682497694917 implements MigrationInterface {
+  name = 'FreeformPost1682497694917';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "post" ADD "content" text`);
+    await queryRunner.query(`ALTER TABLE "post" ADD "contentHtml" text`);
+    await queryRunner.query(
```

Comment:
This is not needed I guess

Reply:
I needed it locally the view needs a update if it exists.
Unless there's an alternative way of refetching it?